packages/evolution-generator/src/scripts/excel_to_csv_generator.py

```
@@ -100,6 +100,13 @@ def write_sheet_to_csv(self, worksheet) -> str:
         print(f"Generated {csv_file_path} successfully")
         return csv_file_path
 
+    @staticmethod
+    def normalize_cell_value(value):
```

Ruff (0.16.2)

[warning] Missing return type annotation for staticmethod `normalize_cell_value`

(ANN205)